be/src/exec/iceberg/iceberg_delete_builder.cpp

```diff
@@ -130,6 +130,51 @@ Status ORCPositionDeleteBuilder::build(const std::string& timezone, const std::s
     }
 }
 
+Status ORCEqualityDeleteBuilder::build(const std::string& timezone, const std::string& delete_file_path,
+                                       int64_t file_length, const std::shared_ptr<HashJoiner> hash_joiner,
+                                       std::vector<SlotDescriptor*> slot_descs, RuntimeState* state) {
+    std::unique_ptr<RandomAccessFile> file;
```

Comment:
Do you think we should allow data cache after this PR

Reply:
pretty nice.